src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -97,21 +114,95 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     toggle: togglePersonalisedSchedule,
   } = useShowHide(false);
 
+  const liveAndFutureEvents = useMemo(
+    () => relatedVenueEvents.filter(isEventLiveOrFuture),
+    [relatedVenueEvents]
+  );
+  const hasSavedEvents = !!liveAndFutureEvents
+    .map(
+      prepareForSchedule({
+        usersEvents: userEventIds,
+      })
+    )
+    .filter((event) => event.isSaved).length;
+
+  const isShowPersonalDownloadBtn = hasSavedEvents && showPersonalisedSchedule;
+  const liveEventsMinimalStartValue = Math.min(
+    ...liveAndFutureEvents.map((event) => event.start_utc_seconds)
+  );
+
+  const firstLiveEvent = liveAndFutureEvents.find(
+    (event) => event.start_utc_seconds === liveEventsMinimalStartValue
+  );
+
+  const minDate = useMemo(
+    () =>
+      firstLiveEvent
+        ? isEventLiveOrFuture(firstLiveEvent)
+          ? new Date(liveEventsMinimalStartValue).getTime()
```

Comment:
use `fromUnixTime` here, please. 
https://date-fns.org/v2.23.0/docs/fromUnixTime

should be something like:
```
fromUnixTime(liveEventsMinimalStartValue)
```

and maybe we should rename liveEventsMinimalStartValue -> liveEventsMinUtcSecs (or Seconds)

Reply:
I've used `getunixtime` there as it's more convenient 58b5cf2